python/cugraph/tests/dask/mg_context.py

```diff
@@ -78,17 +79,18 @@ def _prepare_mg(self):
         self._prepare_comms()
 
     def _prepare_cluster(self):
-        self._cluster = CUDACluster(
-            n_workers=self._number_of_devices,
-            rmm_managed_memory=self._rmm_managed_memory
-        )
+        if self._cluster is not None:
```

Comment:
Shouldn't it create a new CUDA Cluster only if the cluster is None?
```suggestion
        if self._cluster is None:
```

Reply:
Yes, thanks, I realized that just after I pushed this. Fixed in the latest commit.